packages/design-system/src/components/floating-panel/dialog.tsx

```diff
@@ -80,7 +80,8 @@ const contentStyles = css(floatingPanelStyles, {
   top: "50%",
   left: "50%",
   transform: "translate(-50%, -50%)",
-  width: "90vw",
```

Comment:
where is this width coming from?

Reply:
This is from Radix's example. Not sure what to use instead. Feel free to change